airflow/executors/celery_executor.py

```diff
@@ -342,7 +342,12 @@ def sync(self) -> None:
         if self.adopted_task_timeouts:
             self._check_for_stalled_adopted_tasks()
         if time.time() - self.stuck_tasks_last_check_time > self.stuck_queued_task_check_interval:
-            self._clear_stuck_queued_tasks()
+            try:
+                with timeout(seconds=OPERATION_TIMEOUT):
```

Comment:
I'm a little torn on this. Should we just guard the celery query? I'm also not sure we should be silent about it.

The default of 1s short for this whole operation too imo.

I'm not sure we should have this at all, being db backed?

Reply:
I think this check is not so important and timeout error from here especially as we make connection to celery(which is a network request) should be ignored. The 1s is low. I will move the timeout to celery db query